src/anomalib/data/utils/transforms.py

```diff
@@ -46,126 +50,45 @@ def get_transforms(
-        >>> A.save(transforms, "/tmp/transforms.yaml", data_format="yaml")
-        >>> transforms = get_transforms(config="/tmp/transforms.yaml")
-        >>> output = transforms(image=image)
-        >>> output["image"].shape
-        torch.Size([3, 1024, 1024])
+        T.Compose: Torchvision Compose object containing the image transforms.
     """
-    transforms: A.Compose
-
     if config is not None:
-        if isinstance(config, DictConfig):
-            logger.info("Loading transforms from config File")
-            transforms_list = []
-
-            if "Resize" not in config and image_size is not None:
-                resize_height, resize_width = get_image_height_and_width(image_size)
-                transforms_list.append(A.Resize(height=resize_height, width=resize_width, always_apply=True))
-                logger.info("Resize %s added!", (resize_height, resize_width))
-
-            for key, value in config.items():
-                if hasattr(A, key):
-                    transform = getattr(A, key)(**value)
-                    logger.info("Transform %s added!", transform)
-                    transforms_list.append(transform)
-                else:
-                    msg = f"Transformation {key} is not part of albumentations"
-                    raise ValueError(msg)
-
-            transforms_list.append(ToTensorV2())
-            transforms = A.Compose(transforms_list, additional_targets={"image": "image", "depth_image": "image"})
-
-        # load transforms from config file
-        elif isinstance(config, str):
-            logger.info("Reading transforms from Albumentations config file: %s.", config)
-            transforms = A.load(filepath=config, data_format="yaml")
-        elif isinstance(config, A.Compose):
-            logger.info("Transforms loaded from Albumentations Compose object")
-            transforms = config
-        else:
-            msg = "config could be either ``str`` or ``A.Compose``"
-            raise TypeError(msg)
+        # Load torchvision transforms from config
+        pass  # Implement your logic for loading torchvision transforms from a config
     else:
         logger.info("No config file has been provided. Using default transforms.")
         transforms_list = []
 
-        # add resize transform
+        # Add resize transform
         if image_size is None:
-            msg = (
-                "Both config and image_size cannot be `None`. "
-                "Provide either config file to de-serialize transforms or image_size to get the default transformations"
-            )
+            msg = "Both config and image_size cannot be `None`."
             raise ValueError(msg)
         resize_height, resize_width = get_image_height_and_width(image_size)
-        transforms_list.append(A.Resize(height=resize_height, width=resize_width, always_apply=True))
+        transforms_list.append(v2.Resize(size=(resize_height, resize_width), interpolation=3, antialias=True))
```

Comment:
Is there any specific reason why the interpolation mode is set to 3 (Bicubic)? It seems that the default interpolation for the albumentation is `cv2.INTER_LINEAR`?

Reply:
Another comment, to improve code readibilty, maybe it would be better to use the enum `InterpolationMode` instead of using the integer value directly.

Reply:
@djdameln, won't be around for a week to answer, but as far as I remember, interpolation is one of the root cause of not getting good results on some models such as EfficientAD and WinCLIP

Reply:
@samet-akcay Ah, I see. If it is not affecting the speed much, then maybe there should be no problem. It may be less desirable in situations where speed is more important, or the usage of less complex interpolation is enough for the other models to get good results.